cluster/gce/gci/configure-helper.sh

```diff
@@ -366,9 +366,6 @@ function create-master-auth {
   if [[ -n "${KUBE_SCHEDULER_TOKEN:-}" ]]; then
     append_or_replace_prefixed_line "${known_tokens_csv}" "${KUBE_SCHEDULER_TOKEN},"          "system:kube-scheduler,uid:system:kube-scheduler"
   fi
-  if [[ -n "${KUBELET_TOKEN:-}" ]]; then
```

Comment:
is there anything else we can remove related to this, or would that affect other GCE scripts?